core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -390,7 +390,7 @@ export class LoggedOutUser extends BaseUser {
    * Function to click an anchor tag and check if it opens the expected destination
    * in a new tab. Closes the tab afterwards.
    */
-  private async clickLinkAnchorToNewTab(
+  async clickLinkAnchorToNewTab(
```

Comment:
Why do you need this change when the test is not in logged-out-user?

Reply:
Here, I need this function in another class (logged-in user) as well, and couldn't do that as long as the function is private, so removed the private access and moved the function to baseUser (as it is now used by multiple classes so maybe better to keep it in "common") .
Thanks.